tests/__sf_section_time.test.fish

```diff
@@ -0,0 +1,95 @@
+source $DIRNAME/spacefish_test_setup.fish
+
+function setup
+	spacefish_test_setup
+end
+
+test "Time is disabled by default?"
+ 	() = (__sf_section_time)
+end
+
+test "Enabling time! 24-hour by default"
+ 	(
```

Comment:
Looks like the contents of the fishtest parentheses are indented in the other uses.
I would suggest indenting these as well.